contextMenu.less

```diff
@@ -0,0 +1,78 @@
+/*
+ *contextMenu.js v 1.4.0
+ *Author: Sudhanshu Yadav
+ *s-yadav.github.com
+ *Copyright (c) 2013 Sudhanshu Yadav.
+ *Dual licensed under the MIT and GPL licenses
+ *Lessified using http://css2less.co/
+ */
+// Color variables (appears count calculates by raw css)
+@color0: #333333; // Appears 2 times
```

Comment:
All colors are variation of black and white. 
Can we just have two base colors and use lighten or darken method of less to provide variation? This will help a lot on customizing the theme.